README.md

```diff
@@ -22,6 +22,18 @@ EGO_CLIENT_ID=my-ego-application
 EGO_CLIENT_SECRET=secretpassword123
 ```
 
+## Vault
+
+Vault can be used to store your Ego credentials in a secure manner. When Vault is enabled, you will need to provide these env variables:
+```
+VAULT_URL=https://vault.my-vault-instance.com
+VAULT_ROLE=my-policy-name
+VAULT_SECRETS_PATH=/secrets-dir/path-to-secret/my-secret-name
+```
+Your Ego credential keys stored in your secret should match those listed in Ego Authorization.
+
+`VAULT_TOKEN` is optional and only used to login to Vault for local testing.
```

Comment:
Can we call this out as "should not be used in production", "this is available as a tool for developers to test out a configuration only".